docker-compose.override.yml

```diff
@@ -4,6 +4,13 @@ services:
   web:
     volumes:
       - ${APP_ROOT_FOLDER:-.}:/app
+      # do not mount these below local files/folders into the container
+      - /app/public
```

Comment:
Can you add some more info on why this is necessary? I don't all the context.

Reply:
earlier I just mounted the root folder in the local disk as the work directory of the container, so that you don't have to rebuild the container when doing active development.
doing so looks for assets inside the `public/packs` folder, but this might not be present in the local directory without running `assets:precompile`, so am excluding this folder from being mapped and use assets from inside the container.

Reply:
coming to think of it, I could also set the a different target so that the assets are generated/compiled dynamically?
But I am not aware of anyone other than Laura and myself who uses docker to run in locally and decided it would be simpler to do this change